sdk/search/azure-search/src/main/java/com/azure/search/models/TextSplitMode.java

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+
+package com.azure.search.models;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+
+/**
+ * Defines values for TextSplitMode.
+ */
+public enum TextSplitMode {
```

Comment:
String constants instead of enum?

Reply:
Add this to the list of things we need to fix in the Swagger, after this PR is merged.